src/plot_settings.py

```diff
@@ -143,24 +133,38 @@ def get_float(entry):
             "max_right": get_float(self.max_right),
         }
 
-        # Set new item properties
-        if self.style_changed:
-            self.props.application.canvas.apply_limits()
-            pyplot.rcParams.update(file_io.parse_style(
-                plot_styles.get_preferred_style(self.props.application)))
-            for item in self.props.application.datadict.values():
-                item.color = None
-            for item in self.props.application.datadict.values():
-                item.color = \
-                    plotting_tools.get_next_color(self.props.application)
+        graphs.refresh(self.props.application)
+        self.props.application.Clipboard.add()
+        self.props.application.ViewClipboard.add()
+
+    @Gtk.Template.Callback()
```

Comment:
I'm loving the change to do this with the callback, instead of trying to detect if the used style is different from the set style on closing. Apart from the user-experience (it's just nice to see it changing immediately),  it's so much cleaner code-wise as well.